salt/metalk8s/orchestrate/backup/files/job.yaml.j2

```diff
@@ -22,7 +22,10 @@ spec:
       containers:
       - name: backup-replication
         image: {{ image }}
-        command:
+        command: ["/bin/sh", "-c"]
+        args:
+          - ls -t -1 /backups/*.tar.gz | tail -n+6 | xargs rm -f
+          - ;
```

Comment:
I would prefer to do this removal after the `wget`

minor: And I would add the `--verbose` to the `xargs` so that we can actually see from the pod logs what get removed